src/pretix/control/forms/filter.py

```diff
@@ -811,6 +811,211 @@ def meta_properties(self):
         )
 
 
+class OrderPaymentSearchFilterForm(forms.Form):
+    orders = {'id': 'id', 'local_id': 'local_id', 'state': 'state', 'amount': 'amount', 'order': 'order',
+              'created': 'created', 'payment_date': 'payment_date', 'provider': 'provider', 'info': 'info',
+              'fee': 'fee'}
+
+    query = forms.CharField(
+        label=_('Search for…'),
+        widget=forms.TextInput(attrs={
+            'placeholder': _('Search for…'),
+            'autofocus': 'autofocus'
+        }),
+        required=False,
+    )
+    event = forms.ModelChoiceField(
+        label=_('Event'),
+        queryset=Event.objects.none(),
+        required=False,
+        widget=Select2(
+            attrs={
+                'data-model-select2': 'event',
+                'data-select2-url': reverse_lazy('control:events.typeahead'),
+                'data-placeholder': _('All events')
+            }
+        )
+    )
+    organizer = forms.ModelChoiceField(
+        label=_('Organizer'),
+        queryset=Organizer.objects.none(),
+        required=False,
+        empty_label=_('All organizers'),
+        widget=Select2(
+            attrs={
+                'data-model-select2': 'generic',
+                'data-select2-url': reverse_lazy('control:organizers.select2'),
+                'data-placeholder': _('All organizers')
+            }
+        ),
+    )
+    state = forms.ChoiceField(
+        label=_('State'),
+        required=False,
+        choices=[('', _('All payments'))] + list(OrderPayment.PAYMENT_STATES),
+    )
+    provider = forms.ChoiceField(
+        label=_('Payment provider'),
+        choices=[
+            ('', _('All payment providers')),
+        ],
+        required=False,
+    )
+    created_from = forms.DateField(
+        label=_('Ordered from'),
+        required=False,
+        widget=DatePickerWidget,
+    )
+    created_until = forms.DateField(
+        label=_('Ordered until'),
```

Comment:
These are incorrectly labeled, "Ordered from" would be ``order.datetime``, not ``payment.created``. The correct label would be "Payment created from/until"